src/components/CourseCatalog.jsx

```diff
@@ -0,0 +1,86 @@
+import {React, useState, useEffect} from 'react';
+import {Card, Modal, Button} from 'react-bootstrap';
+import Cookies from 'universal-cookie';
+
+function getCourses() {
+  const cookies = new Cookies();
+  const baseUrl = process.env.API_BASE_URL || 'http://localhost:8000';
+  return fetch(`${baseUrl}/api/courses`, {
+    headers: {
+      'Content-Type': 'application/json',
+      'Authorization': `Token ${cookies.get('token')}`,
+    },
+    method: 'GET',
+  })
+      .then((data) => data.json())
+      .catch((err)=>console.log(err));
+}
+function takeCourse(studentId, courseCode) {
+  const cookies = new Cookies();
+  const baseUrl = process.env.API_BASE_URL || 'http://localhost:8000';
+  return fetch(`${baseUrl}/api/students/
+  ${studentId}/takeCourse/${courseCode}`, {
+    headers: {
+      'Content-Type': 'application/json',
+      'Authorization': `Token ${cookies.get('token')}`,
+    },
+    method: 'PUT',
+  })
+      .then((data) => data.json())
+      .catch((err)=>console.log(err));
+}
+const CourseCatalog = ({studentId}) => {
+  const [courses, setCourses] = useState([]);
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  useEffect(()=>{
+    getCourses()
+        .then((data) => {
+          setCourses(data);
+        },
+        )
```

Comment:
what's with the comma?
```suggestion
        })
```